npm/pkg/transport/consts.go

```diff
@@ -0,0 +1,7 @@
+package transport
+
+const (
+	// concurrentInputRegistrations = 10
+	grpcMaxConcurrentStreams = 100
+	remoteAddrContextKey     = "remote-addr"
```

Comment:
seems like this is used as a key to fetch context values. usually these are their own unexported types in order to avoid key collision in the global scope.

Reply:
Done.